tools/scripts/aducm.mk

```diff
@@ -223,9 +223,35 @@ $(MQTT_LIB):
 
 endif
 
+#	IIO
```

Comment:
Question: this assumes a project treats the IIO as sources instead of a library and changes that. If we pass this change should the projects treat iio as a library directly instead of sources?

Reply:
If libraries/iio is found in src_dirs will use libraries/iio library. 
I am sure other aproach will fit better.
But for now, for consistency with the rest of the makefile I prefere to keep them this way and in an other PR  use an other approach for the rest of the libraries too.